packages/cocoapods-cache-plugin/lib/cocoapods_plugin.rb

```diff
@@ -0,0 +1,79 @@
+require 'cocoapods-cache-plugin/command'
+
+CDN_URL = "https://cdn.cocoapods.org"
+
+POD_BLACKLIST = ["libwebp", "Braintree"]
+
+COCOAPODS_CACHE_URL = ENV['COCOAPODS_CACHE_URL']
+
+module Pod
+  class Installer
+    class Analyzer
+      @@_was_using_cocoapods_cache_printed = false
+
+      alias_method :orig_sources, :sources
+
+      # add our own source to the list of sources
+      def sources
+        if COCOAPODS_CACHE_URL
```

Comment:
```suggestion
        if NEXUS_INSTANCE_URL
```

Reply:
Maybe 
```suggestion
        if NEXUS_COCOAPODS_REPO_URL
```
?
In my opinion, `NEXUS_INSTANCE_URL` name may encourage people to use `http://10.254.24.6` URL vs `http://10.254.24.6/repositories/cocoapods-repo`. What do you think?